rsconcept/frontend/src/features/rsmodel/labels.ts

```
@@ -117,3 +118,191 @@ export function printTypeCrumbs(type: RO<Typification>, path: TypePath, index: n
+        break;
+    }
+  }
+  return false;
+}
+
+function prepareValueInternal(value: Value, type: ExpressionType, schema: RSForm, dataContext: BasicsContext): Doc {
+  switch (type.typeID) {
+    case TypeID.integer:
+      return text(String(value));
+    case TypeID.basic:
+      const cst = schema.cstByAlias.get(type.baseID);
```

Biome (2.4.15)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)